### ISSUE TITLE
Inconsistency between code and comments for a sort

### ISSUE THREAD

**Issue by Exter-N** [TestRunnerSRL/OoT-Randomizer#423]
https://github.com/TestRunnerSRL/OoT-Randomizer/blob/50ca50a525980d835b6a2837e44d005596710149/Fill.py#L142-L145
The comment says "Boss Key, Small Key, Other", but the sort is in ascending order so it's actually "Other, Small Key, Boss Key". Is the comment outdated or is it a (subtle) bug in the code?

**Comment by TestRunnerSRL** [comment]
It is more likely that this is backwards. It should be bosskeys first. This is something I kinda debate about since placing bosskeys before smallkeys causes placement failure issues. On the other hand it's argued that having the boss key more "random" is more important.

**Comment by Exter-N** [comment]
What do you think about using a BK then SK then Other ordering, and a multiple-attempt algorithm like the one used for the shops and songs to work around placement failure issues?

**Comment by Exter-N** [comment]
Yup, just tried, and indeed it's iterated backwards… I guess I got misled by the comments.

Do you accept PRs that just add more detailed comments? To prevent other newcomers from making the same mistakes as me xD

**Comment by TestRunnerSRL** [comment]
yeah absolutely. I think there there's a severe lack of comments in general. I made sure to comment up `fill.py` because it was one of the more confusing files.
